test/uk/gov/hmrc/tai/util/viewHelpers/TaiViewSpec.scala

```diff
@@ -127,6 +127,7 @@ trait TaiViewSpec extends BaseSpec with JsoupMatchers {
   def pageWithContinueButtonFormNew(submitUrl: String): Unit =
     pageWithButtonFormNew(submitUrl, "Continue")
 
+//  TODO: This function is unused - Remove?
```

Comment:
do/ should  we replace the old button id with new id  enable test for new button

Reply:
As far as I'm aware we don't use this function though

Reply:
and the new button doesn't have an id

Reply:
Deleted this function as not used